be/src/exec/pipeline/sink/sink_io_buffer.h

```diff
@@ -84,9 +84,9 @@ class SinkIOBuffer {
     virtual void cancel_one_sinker() { _is_cancelled = true; }
 
     virtual void close(RuntimeState* state) {
-        _is_finished = true;
```

Comment:
some other operators implement specific sink buffer based on SinkIOBuffer, e.g. export_sink_operator. if we change here, other operators' implementation should also change